app/services/audit_results.rb

```diff
@@ -222,7 +222,11 @@ def send_honeybadger_notification(result)
   end
 
   def events_client
-    Dor::Services::Client.object(druid).events
+    Dor::Services::Client.object(namespaced_druid).events
+  end
+
+  def namespaced_druid
+    druid.start_with?('druid:') ? druid : "druid:#{druid}"
```

Comment:
Do we need to have this contitional?  Can we just define the interface as accepting one or the other?

Reply:
so would the preference be to modify the dor-services-client code to accept both, and then bump the version of DSC that pres cat uses?

i'd be fine with that.  though i'd also have a slight preference for merging this in the meantime and doing that as follow on work, since this is done, and fixes a production bug.

Reply:
No, I'm suggesting that `AuditResults` should pick a way and use it and not accept both.

Reply:
ah, gotcha, apologies for misunderstanding.

yeah, so now that you mention it, preservation catalog does consistently use the unprefixed version of the druid internally, throughout the code and in the DB.  so the conditional is actually superfluous safeguarding.  i wasn't thinking when i suggested that approach in #1388, and then i copypasta'd my own bad suggestion.  happy to remove.

thanks for the merge despite the rough edge!  follow on coming shortly...

Reply:
addressed in https://github.com/sul-dlss/preservation_catalog/pull/1418